api.lua

```diff
@@ -467,15 +467,6 @@ signs_lib.charwidth_wide32 = build_char_db(32)
 
 local math_max = math.max
 
-local function fill_line(x, y, w, c, font_size, colorbgw)
-	c = c or "0"
-	local tex = { }
-	for xx = 0, math.max(0, w), colorbgw do
-		table.insert(tex, (":%d,%d=signs_lib_color_"..font_size.."px_%s.png"):format(x + xx, y, c))
```

Comment:
Aren't the `signs_lib_color` files unused now that you switched this to a color lookup table & `^[colorize`? Can't you remove them now (or is something else using them)?